From 4fc52fc404489953ff733e757a98844e17c7a1b1 Mon Sep 17 00:00:00 2001
From: Markus Volk <flk@linux.fritz.box>
Date: Mon, 17 Feb 2020 08:50:16 +0100
Subject: [PATCH] fix

---
 data/y-web/Y_Settings_yWeb.yhtm | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/data/y-web/Y_Settings_yWeb.yhtm b/data/y-web/Y_Settings_yWeb.yhtm
index fef7829a8..37ca2f5fa 100644
--- a/data/y-web/Y_Settings_yWeb.yhtm
+++ b/data/y-web/Y_Settings_yWeb.yhtm
@@ -137,8 +137,8 @@ select {
 						<option value="10">Mutant | HD51</option>
 						<option value="11">WWIO | Bre2ze 4K</option>
 						<option value="12">VU+ | Solo 4K</option>
-						<option value="13">Zgemma | H7</option>
-						<option value="14">Edision | OSMIO+ 4K</option>
+						<option value="18">Zgemma | H7</option>
+						<option value="19">Edision | OSMIO+ 4K</option>
 					</select>
 				</td>
 			</tr>
